lib/datasource/remote/api/proposals_repository.dart

```diff
@@ -253,8 +254,9 @@ class ProposalsRepository extends NetworkRepository with EosRepository {
     return http
         .post(url, headers: headers, body: request)
```

Comment:
Pet peeve of mine that we always return untyped futures, just generic "result" - I really want typed results. 

We can probably do this with some Dart magic right @RaulUrtecho @gguijarro-c-chwy 

Bit of a side comment but I want this to read (for example)

```Future<Result<List<DelegatorModel>>> ```

Reply:
That's not a change request, that would go into another PR, so please ignore